bindings/src/account_history/types/coin_value.rs

```diff
@@ -10,6 +10,14 @@ pub struct CoinValue {
     pub amount_usd: Decimal,
 }
 
+#[cw_serde]
+pub struct DecCoinValue {
```

Comment:
Here we can derive and Default trait to remove the explicit **impl Default**

```suggestion
#[derive(Default)]
pub struct DecCoinValue {
```